src/lib.rs

```diff
@@ -233,6 +238,15 @@ pub struct Stream {
 /// A builder for calling the `CreateCoreWebView2EnvironmentWithDetails`
 /// function.
 #[derive(Default)]
+#[cfg(feature = "embed-dll")]
+pub struct EnvironmentBuilder<'a> {
+    browser_executable_folder: Option<&'a Path>,
+    user_data_folder: Option<&'a Path>,
+    additional_browser_arguments: Option<&'a str>,
+}
+
+#[derive(Default)]
+#[cfg(not(feature = "embed-dll"))]
```

Comment:
You don't need to duplicate the struct definition. The attribute can be applied to the `dll_file_path` field.